graphitron-codegen-parent/graphitron-java-codegen/src/test/java/no/sikt/graphitron/queries/fetch/OutputTest.java

```diff
@@ -166,12 +170,24 @@ void outerNestedRow() {
     void outerNestedListedRow() {
         assertGeneratedContentContains(
                 "outerNestedListedRow", Set.of(CUSTOMER_TABLE),
-                ".row(DSL.row(DSL.multiset(DSL.select(DSL.row(_a_customer.getId()).mapping(Functions.nullOnAllNull(CustomerTable::new)))" +
-                        ".from(_a_customer)" +
-                        ".orderBy(_a_customer.fields(_a_customer.getPrimaryKey().getFieldsArray()))))" +
-                        ".mapping(_iv_e -> _iv_e.map(Record1::value1)))" +
-                        ".mapping(Functions.nullOnAllNull((_iv_it) -> new Wrapper(_iv_it))))" +
-                        ".fetchOne(_iv_it -> _iv_it.into(Wrapper.class))"
+                ".select(queryForQuery_wrapper())",
+                ".fetchOne(_iv_it -> _iv_it.into(Wrapper.class))",
+                // Helper uses multiset for list
+                """
+                private static SelectField<Wrapper> queryForQuery_wrapper() {
+                    var _a_customer = CUSTOMER.as("customer_2168032777");
+                    return DSL.row(
+                                    DSL.row(
+                                            DSL.multiset(
+                                                    DSL.select(queryForQuery_wrapper_d1_customer())
+                                                    .from(_a_customer)
+                                                    .orderBy(_a_customer.fields(_a_customer.getPrimaryKey().getFieldsArray()))
+                                            )
+                                    ).mapping(_iv_e -> _iv_e.map(Record1::value1))
+                            ).mapping(Functions.nullOnAllNull((_iv_it) -> new Wrapper(_iv_it)));
+                }
+                """,
+                "private static SelectField<CustomerTable> queryForQuery_wrapper_d1_customer() {"
```

Comment:
Forenkling:

```suggestion
                ".row(DSL.row(DSL.multiset(DSL.select(queryForQuery_wrapper_d1_customer())" +
                ".from(_a_customer).orderBy(_a_customer.fields(_a_customer.getPrimaryKey().getFieldsArray())))" +
                ").mapping(_iv_e -> _iv_e.map(Record1::value1))" +
                ").mapping(Functions.nullOnAllNull((_iv_it) -> new Wrapper(_iv_it)));"
```

Litt samme som over. `.fetchOne(_iv_it -> _iv_it.into(Wrapper.class))` er ikke med, fordi den var egentlig bare der for se at koden slutter der og at det ikke er noe mer etterpå. Selve `queryForQuery_wrapper()` spørringer er ikke interessant her.